app/assets/stylesheets/components/page_specific/_promo.scss

```diff
@@ -9,19 +9,73 @@
 .promo__heading {
   @extend %font-heading-medium;
   @include body(18, 24);
-  margin-top: 0;
+  margin: 0 0 $baseline-unit;
```

Comment:
I think our convention (@benpbarnett) is to avoid using only 3 options like this. My preference is to declare it explicitly `margin-bottom: 0`.